wagtail/admin/templates/wagtailadmin/shared/field.html

```diff
@@ -57,7 +57,7 @@
         </div>
 
         {# Separate container for the widget with prefix icon and suffix comment button #}
-        <div class="w-field__input" data-field-input>
+        <div class="w-field__input" data-field-input {% for name, value in attrs.items %}{% if value is not False %} {{ name }}{% if value is not True %}="{{ value|stringformat:'s' }}"{% endif %}{% endif %}{% endfor %}>
```

Comment:
Firstly - I think the requirements are to put these `attrs` on the root div - which would be line 17 in this file, a reminder we are replacing the `data-field-wrapper` attribute with the BASE_ATTRS approach, so where that attribute is is where our new code should go.

```suggestion
<div class="w-field__wrapper {{ classname }}" {% include "wagtailadmin/tables/attrs.html" with attrs=self.attrs %}>
```

Let's use this template `wagtail/admin/templates/wagtailadmin/tables/attrs.html`, but we probably want to move this template to somewhere shared.

Here is a usage example `wagtail/admin/templates/wagtailadmin/tables/title_cell.html`

With the template moved to the shared folder (`wagtail/admin/templates/wagtailadmin/shared/attrs.html`, it should look something like this...

```suggestion
<div class="w-field__wrapper {{ classname }}" {% include "wagtailadmin/shared/attrs.html" with attrs=self.attrs %}>
```

Important: I got this working but it has to use the `self.attrs` not just `attrs` - we may need some help from Matt or others to see if we can improve this but it should be good enough for you to continue your dev and start on tests.